### ISSUE TITLE
[BUG] Node version in Playwright docker image mcr.microsoft.com/playwright:focal is v14 instead of v16

### ISSUE THREAD

**Issue by TimeInvestor** [microsoft/playwright#10007]
In the Dockerfile (https://github.com/microsoft/playwright/blob/master/utils/docker/Dockerfile.focal) it states
```sh
# Install node16
RUN apt-get update && apt-get install -y curl && \
    curl -sL https://deb.nodesource.com/setup_16.x | bash - && \
    apt-get install -y nodejs
```
However,  pulled and ran the docker image by
```sh
docker run -it --rm --ipc=host mcr.microsoft.com/playwright:focal /bin/bash
```
It gives Node 14 inside the container:
```sh
root@103021ccc778:/# node -v
v14.18.1
```

**Comment by mxschmitt** [comment]
We recently changed it to Node.js 16, since 16 is the new LTS. It gets released in 1.17.

For 1.16 its still Node.js 14, see here: https://github.com/microsoft/playwright/blob/release-1.16/utils/docker/Dockerfile.focal

**Comment by TimeInvestor** [comment]
Got it.
Thanks for the explanation.
